src/screens/LearningScreen/components/FlashCardContainer.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import { View, Button, Text } from 'react-native';
+
+import { getUserData } from '../../../utils/storage';
+import {
+  getRandomWord,
+  handleCorrectOrIncorrect,
+  getNewLevel,
+} from '../../../utils/spacing';
+import FlashCard from './FlashCard';
+
+const handleRevealButtonPress = (setRevealed) => {
+  setRevealed((flipped) => !flipped);
+};
+
+const handleExpressionsButtonPress = (setExpanded) => {
+  setExpanded((isExpanded) => !isExpanded);
+};
+
+const FlashCardContainer = () => {
+  const [list, setList] = useState({});
+  const [wordInfo, setWordInfo] = useState({});
+  const [done, setDone] = useState(false);
+  const [isExpanded, setExpanded] = useState(false);
+  const [isRevealed, setRevealed] = useState(false);
+  const [currentLevel, setCurrentLevel] = useState('one');
```

Comment:
Just curious about this value — why use `one` and not a number like `0` for the level?

Reply:
That's a byproduct of the way I set up my data. I do think now that it would make more sense to make it a number, and I may change that eventually but it'll take a bit of effort